code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -2144,3 +2438,14 @@
 	glass_name = "special blend whiskey"
 	glass_desc = "Just when you thought regular station whiskey was good... This silky, amber goodness has to come along and ruin everything."
 	glass_center_of_mass = list("x"=16, "y"=12)
+
+/datum/reagent/ethanol/specialwhiskey/affect_ingest(var/mob/living/carbon/M, var/alien, var/effect_multiplier)
+    ..()
+    M.add_chemical_effect(CE_PAINKILLER, 50)
+
```

Comment:
That's tons of newlines.

Reply:
Wait, they are spaces not tabs behind ..() and M.add.....

Reply:
Thanks for that ill do that

Reply:
![image](https://user-images.githubusercontent.com/36856516/75633354-08f8bc00-5bd2-11ea-9fb9-eb929309e4f3.png)
Here's from Notepad++

Reply:
Okay everything should work now